internal/rbd/clone.go

```diff
@@ -218,62 +199,22 @@ func (rv *rbdVolume) doSnapClone(ctx context.Context, parentVol *rbdVolume) erro
 			}
 		}
 
-		if err != nil || errFlatten != nil {
-			if !errors.Is(errFlatten, ErrFlattenInProgress) {
-				// cleanup snapshot
-				cErr := cleanUpSnapshot(ctx, parentVol, tempSnap, tempClone)
-				if cErr != nil {
-					log.ErrorLog(ctx, "failed to cleanup image %s or snapshot %s: %v", tempSnap, tempClone, cErr)
-				}
+		if err != nil {
+			// cleanup snapshot
+			cErr := cleanUpSnapshot(ctx, parentVol, tempSnap, tempClone)
+			if cErr != nil {
+				log.ErrorLog(ctx, "failed to cleanup image %s or snapshot %s: %v", tempSnap, tempClone, cErr)
```

Comment:
```suggestion
				log.ErrorLog(ctx, "failed to cleanup image %s or snapshot %s: %v",  tempClone,tempSnap, cErr)
```
not related to this PR but can you correct this logging.

Reply:
done